analytics.js

```diff
@@ -297,6 +305,15 @@ var Analytics = {
                     result.totals.ie_version[version] += parseInt(result.data[i].visits);
                 }
             }
+            if (_.startsWith(report.name, "top-countries")) {
+                // Extract the total visits from the `United States`
+                for (var i=0; i < result.data.length; i++) {
+                    if (result.data[i].country === "United States") {
+                        result.totals.us_visits = result.data[i].visits;
+                    }
```

Comment:
I think this is unnecessary, since it's only grabbing one value with a predictable name, rather than doing a subtotal of some sort. I'll remove it before merge.